agot-bg-game-server/src/common/ingame-game-state/action-game-state/resolve-raid-order-game-state/resolve-single-raid-order-game-state/ResolveSingleRaidOrderGameState.ts

```diff
@@ -92,9 +92,18 @@ export default class ResolveSingleRaidOrderGameState extends GameState<ResolveRa
                 throw new Error();
             }
 
+            let raiderGainedPowerToken: boolean | null = null;
+            let raidedHouseLostPowerToken: boolean | null = null;
+
             if (orderTarget.type instanceof ConsolidatePowerOrderType) {
+                const raiderOldPt = this.house.powerTokens;
+                const raidedOldPt = this.house.powerTokens;
+
                 this.ingameGameState.changePowerTokens(this.house, 1);
                 this.ingameGameState.changePowerTokens(raidedHouse, -1);
+
+                raiderGainedPowerToken = raiderOldPt != this.house.powerTokens;
+                raidedHouseLostPowerToken = raidedOldPt != raidedHouse.powerTokens;
```

Comment:
`changePowerTokens` returns the true delta of Power tokens. It could also be used to determine whether the house trully gained a PT.

It's okay that way, but I tell it here so that you know.

Reply:
Ah no... I didn't knew it. Let me change that quickly.